contrib/debian/control

```diff
@@ -0,0 +1,23 @@
+Source: netdata
+Build-Depends: debhelper (>= 9),
+ dpkg-dev (>= 1.13.19),
+ dh-systemd (>= 1.5),
+ zlib1g-dev,
+ autoconf,
+ automake,
+ pkg-config
+Section: net
+Priority: optional
+Maintainer: Costa Tsaousis <costa@tsaousis.gr>
```

Comment:
@lhw thanks for the comments!

You're right on all of it, but I think we're looking at the packaging from two different perspectives.

If you're building a debian/ directory for the Debian project, then yes add the other dependencies etc and split up into multiple packages. You can deprecate older systems because you won't generally need to build for them.

But this is putting together packaging so that someone can take the upstream source, hopefully on pretty much any recent .deb based system, and build something that works. That's a different goal (and why we moved it into contrib/).

So although I haven't tested it, hopefully you should be able to download netdata source onto Ubuntu whatever, or Mint, or other systems and easily build a recent package for your system. Adding in all the Debian dependencies or strictly following Debian Policy on this doesn't help them, as their systems are likely to be different. This packaging should be as simple and universal as possible.

That's also why source/format is "native". Non-native means someone who checks out from git then has to build a tarball (following a precise naming scheme) in the parent directory before the build will even commence. Whereas "native" is perfect for a non-official package - the build will complete for them and they don't need to care about generating a diff (which is empty anyway).

Does that make sense?

However - very happy if you want to take this as a basis for an official Debian package, host it on the Debian systems as you normally would and then of course it would be put in the standard debian/ location as you build.

Having said that, "wrap-and-sort" - cool :-) not come across that before!

Cheers,
Matthew